packages/internal/identity-resolver/src/identity-resolver.ts

```diff
@@ -13,14 +13,19 @@ import {
 import { IdentityResolverError } from './identity-resolver-error'
 import { asNormalizedHandle, extractNormalizedHandle } from './util'
 
-export type ResolvedIdentity = {
-  document: DidDocument<AtprotoIdentityDidMethods>
```

Comment:
I wonder if moving to `didDoc` also comes with the small benefit of not conflicting with the browser global `document` 👍